src/omnicore/tx.cpp

```diff
@@ -82,6 +82,16 @@ bool CMPTransaction::isOverrun(const char* p)
 
 // -------------------- PACKET PARSING -----------------------
 
+/** Temporarily disables packet debugging and then calls interpret_Transaction(). */
+bool CMPTransaction::reinterpret_Transaction()
+{
+    bool packetDebugState = msc_debug_parser;
+    msc_debug_parser = 0;
```

Comment:
As far as I can see this has no effect currently, because `msc_debug_parser` isn't the flag used to log packet parsing, but `msc_debug_packets`.